paymentintent.go

```diff
@@ -313,12 +360,25 @@ type PaymentIntentNextActionRedirectToURL struct {
 	URL       string `json:"url"`
 }
 
+// PaymentIntentNextActionUseStripeSDK represents the resource for the next action of typee "use_stripe_sdk".
+type PaymentIntentNextActionUseStripeSDK struct {
+}
```

Comment:
Is adding this one expected? Not opposed since it's really a thing but I didn't expect it

Reply:
Oh yes, forgot to call that out. Codegen revealed it was missing so I figured I'd add it alongside `verify_with_microdeposits`.